concourse/pipelines/model/traits/slack.py

```diff
@@ -32,19 +53,39 @@ def _defaults_dict(self):
 
     def _optional_attributes(self):
         return {
-            'slack_channel',
-            'slack_cfg_name',
+            'default_channel',
```

Comment:
should this not be a required attribute?

Reply:
right